docker/neumann/Dockerfile

```
@@ -33,7 +37,11 @@ USER $USERNAME
 
 # Copy files
 COPY --chown=$USERNAME --from=builder /build/target/release/$BINARY  /$USERNAME/$BINARY
-COPY --chown=$USERNAME ./resources /$USERNAME/resources
+RUN mkdir /$USERNAME/resources
+COPY --chown=$USERNAME --from=builder /build/neumann-dev-chain-spec.json  /$USERNAME/resources/neumann-dev-chain-spec.json
+COPY --chown=$USERNAME --from=builder /build/neumann-local-chain-spec.json  /$USERNAME/resources/neumann-local-chain-spec.json
+COPY --chown=$USERNAME --from=builder /build/neumann-staging-chain-spec.json  /$USERNAME/resources/neumann-staging-chain-spec.json
+COPY --chown=$USERNAME --from=builder /build/neumann-latest-chain-spec.json  /$USERNAME/resources/neumann-latest-chain-spec.json
```

Haven't tested this, but I think you can copy all of it in one command and only create one layer.
```suggestion
COPY --chown=$USERNAME --from=builder /build/neumann-dev-chain-spec.json /build/neumann-local-chain-spec.json /build/neumann-staging-chain-spec.json /build/neumann-latest-chain-spec.json /$USERNAME/resources
```